scripts/travis/upload-to-docker.sh

```diff
@@ -31,15 +31,6 @@ if [[ -n $major ]]; then
   fi
```

Comment:
do you know the meaning of substitution in L19?

Reply:
Not sure, but I will try it and update the comment accordingly.

Reply:
I think long back BRANCH was evaluated as `/refs/head/branch-name` and hence it was done to remove `/` from branch name since that will result in docker tag:

```
$ a="/refs/head/foobar"
$ echo ${a///}
refsheadfoobar
```

And `/` isn't allowed in docker tag:

```
$ docker tag jaegertracing/jaeger-ingester jaegertracing/jaeger-ingester:testing/testing
Error parsing reference: "jaegertracing/jaeger-ingester:testing/testing" is not a valid repository/tag: invalid reference format
$ docker tag jaegertracing/jaeger-ingester jaegertracing/jaeger-ingester:testing
$
```